data/notebooks/Export_Table_ACLs.py

```diff
@@ -0,0 +1,177 @@
+# Databricks notebook source
+# MAGIC %md #Export Table ACLs
+# MAGIC 
+# MAGIC Exports Table ACLS to a file on DBFS, which can be imported using the Import_Table_ACLs script
+# MAGIC 
+# MAGIC Parameters:
+# MAGIC - Databases: [Optional] comma separated list of databases to be exported, if empty all databases will be exported
+# MAGIC - OutputPath: Path to write the exported file to (Choose a something under DBFS:/user/hive/warehouse, there an admin has write access)
+# MAGIC - OutputFormat: JSON
+# MAGIC 
+# MAGIC Execution: Run the notebook on a cluster with Table ACL's enabled as a user who is an admin 
+# MAGIC 
+# MAGIC Supportes ACLs for Object types:
+# MAGIC - Catalog: included if all databases are exported, not included if databases to be exported are specified
+# MAGIC   - Database: included
+# MAGIC     - Table: included
+# MAGIC     - View: included
+# MAGIC     -
+# MAGIC 
+# MAGIC WARNING: Needs more testing and fixing ... so far all I know with certainty is, that it does not work quite right yet
+# MAGIC 
+# MAGIC tomi.schumacher@databricks.com
```

Comment:
Let's manually delete any reference to your email here. Both notebooks you can safely delete your email address.